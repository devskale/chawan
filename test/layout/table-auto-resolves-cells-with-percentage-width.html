<!DOCTYPE html>
<table>
<tr>
<td width="20%"></td>
<td>Lorem ipsum dolor sit amet, in pri debet blandit, ius an idque voluptaria, vidit phaedrum definitionem his ne.</td>
<td width="20%"></td>
</table>
