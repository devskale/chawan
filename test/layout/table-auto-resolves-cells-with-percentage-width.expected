                 Lorem ipsum dolor sit amet, in pri debet
                 blandit, ius an idque voluptaria, vidit
                 phaedrum definitionem his ne.
